x-pack/plugins/enterprise_search/public/applications/app_search/components/result/types.ts

```diff
@@ -25,18 +25,6 @@ export interface ResultMeta {
   clicks?: number;
 }
 
-// A search result item
-export type Result = {
-  id: {
-    raw: string;
-  };
-  _meta: ResultMeta;
-} & {
-  // this should be a FieldType object, but there's no good way to do that in TS: https://github.com/microsoft/TypeScript/issues/17867
-  // You'll need to cast it to FieldValue whenever you use it.
-  [key: string]: object;
-};
-
```

Comment:
This type was replaced with the SearchResult type from Search UI.
It doesn't have info about id and _meta fields — I created [an issue](https://github.com/elastic/enterprise-search-team/issues/2226) to investigate whether it'll always be present.